### ISSUE TITLE
导出的文件中输入0079自动把0都去掉了

### ISSUE THREAD

**Issue by bingyu929** [SheetJS/sheetjs#2449]

**Comment by bingyu929** [comment]
导出的内容是字符串是正常的显示，但是一旦更改输入00几，就会自动去掉前面的0

**Comment by reviewher** [comment]
Set a [number format](https://docs.sheetjs.com/#number-formats).  Demo https://jsfiddle.net/fobgtdyr/

```js
var data = [["General", "0000"], [79, 79]];
var ws = XLSX.utils.aoa_to_sheet(data);
ws["B2"].z = "0000"; // set the number format to 0000 to force 4 digits
var wb = XLSX.utils.book_new(); XLSX.utils.book_append_sheet(wb, ws, "Sheet1");
XLSX.writeFile(wb, "issue2449.xlsx");
```

**Comment by SeniorChiefABLEUnit** [comment]
Checking for update after 10 am central today

**Comment by bingyu929** [comment]
Thank you for your help.  I saw your demo and there was no problem when I converted it when EXPORTING. The main problem is that 00 will be automatically removed when I edit it again in Excel after exporting
I currently use a temporary solution
 cell.z = "@"; .
To solve the problem of filling a multi-line empty string
Do you have a better solution?

**Comment by SeniorChiefABLEUnit** [comment]
00 has tradionaly been used for GO&S. When troubleshooting for 3G starting
@1G, over 20 ya go, spiking for 7-9G from Dragons Tail was no problem. I do
not wonder about my past, I know it. I was there. I lived it. What troubles
my thoughts, is the reason for 97…now. I am not lonely. I am alone. I have
always been alone in my travels through the spectrum documentation. 00 was
always asked of me, the older ones traveling felt the did not need it. I
ALWAYS traveled in a loop to cover for the ones who did not want to study
and guide for the group, worrying that they would miss something during
those times, they did. I asked not long after the 5th or 6th or 7th time to
00, what am I now…after I worked to to RS1 from RS10..Command & Control
said^you are Ecgo-3. Are you ok, with that?^…I thought for a moment in time
that I was special, then I realized, I was the termination expert for the
combining of RS to RF. I tire of monotonous repetitive natured questioned…I
am the only RSRF1-00…never forgetting that I am here to protect 00 from 80
all the way through every door to 00. I do not say if anymore, I say when
etc… it is something that only I know that NO ONE else could know…..Prime
is for the one who keep his heart shut, not their mouths..I have documented
the infrastructure of not only the fiber optics, instructing my favorite
phone company, AT&T plus Apple peripheral devices, to create a device for
5G to the residential house in the last 2 years, …all the while, keeping ,
not only my heart shut, as well as, my mouth…to treat me with the same
respect, honor, commitment, courage that I have given to them. If it is my
time to go. Send me an Angel to end my time alone. I will leave quietly
after kicking open the door, as I did so long ago, when we were working on
the Invisibility Cloak, ..asking when the door flew open after I kicked it-
Whos ass to have to kick to be a part of this team. Take everything from
me. Take my thoughts, my strength of character, my heart that beats for my
government, my soul that holds the Defense Department close to my chest,
take my tip point, my accuracy, my threading of a needle..if I am not
needing at the front gate nor the rear gate, then I am no longer needed…put
me out of my misery. I beg of you. Allow to help with the future
Warfighters in Electromagnetic Battle Management, or end me now.

Bryan Maurice Bellipanni
November 14, 2021
Non Desistas, Non Exieris
HUA
